packages/components/popper/src/content.vue

```diff
@@ -218,12 +226,12 @@ onMounted(() => {
       if (isElement(el)) {
         const { ariaLabel, id } = toRefs(props)
         triggerTargetAriaStopWatch = watch(
-          [role, ariaLabel, ariaModal, id],
+          [role, ref(ariaLabel), ariaModal, ref(id)],
```

Comment:
I do not quite understand why `ref(xx)` is used.

Reply:
Because `ariaLabel ` and `id` may be `undefined`, this array is an invalid watch source. Although the result is not wrong, it will cause ts type errors.

Reply:
like this:
<img width="994" alt="WX20220703-131325@2x" src="https://user-images.githubusercontent.com/23100055/177025877-7347ca0b-ba25-44ad-83fa-264ed99f322c.png">

Reply:
I think it maybe caused by the `toRefs`

```ts
const { ariaLabel, id } = toRefs(props)
// id: Ref<string | undefined> | undefined
// props.id: string | undefined
```

Reply:
maybe change it to:
```ts
watch(
  [role, () => props.ariaLabel, ariaModal, () => props.id],
  (watches) => {
    ...
  },
  { immediate: true }
)
```
what do you think?

Reply:
This should work fine

Reply:
updated.